tools/scripts/os.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then # Linux
```

Comment:
Depending on the OS use the appropriate command.  I could have simplified this, but decided to keep all the different options.